modules/updating-a-connected-data-source.adoc

```diff
@@ -19,18 +19,19 @@ endif::[]
 .Procedure
 . From the {productname-short} dashboard, click *Data Science Projects*.
 +
-The *Data science projects* page opens.
+The *Data Science Projects* page opens.
 . Click the name of the project whose data source you want to change.
 +
 A project details page opens.
-. Click the action menu (*&#8942;*) beside the data source that you want to change in the *Data connections* section and click *Change connected workbenches*.
+. Click the *Data connections* tab.
+. Click the action menu (*&#8942;*) beside the data source that you want to change and then click *Change connected workbenches*.
```

Comment:
Looks like this section needs to be:

'and then click *Edit data connection*.
+
The *Edit data connection* dialog opens.
. In the *Connected workbench* field, select a workbench to connect.
. Click *Update data connection*.'

Reply:
Good catch, thank you! Updated now!